client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala

```diff
@@ -49,14 +56,144 @@ object CapiModelEnrichment {
         isMedia -> Media,
         isReview -> Review,
         tagExistsWithId("tone/analysis") -> Analysis,
-        isComment -> Comment,
+        isCommentDesign -> Comment,
         tagExistsWithId("tone/features") -> Feature,
-        liveBlog -> Live,
-        deadBlog -> Article
+        isLiveBlog -> Live,
+        isDeadBlog -> Article
       )
 
-      val result = predicates.collectFirst { case (predicate, design) if predicate(content) => design }
+      val result = getFromPredicate(content, predicates)
       result.getOrElse(defaultDesignType)
     }
   }
+
+  implicit class RenderingFormat(val content: Content) extends AnyVal {
+
+
+    def design: Design = {
+
+      val defaultDesign: Design = ArticleDesign
+
+      val isPhotoEssay: ContentFilter = content => content.fields.flatMap(_.displayHint).contains("photoessay")
+
+      val predicates: List[(ContentFilter, Design)] = List(
+        tagExistsWithId("artanddesign/series/guardian-print-shop") -> PrintShopDesign,
+        isMedia -> MediaDesign,
+        isReview -> ReviewDesign,
+        tagExistsWithId("tone/analysis") -> AnalysisDesign,
+        isCommentDesign -> CommentDesign,
+        tagExistsWithId("tone/features") -> FeatureDesign,
```

Comment:
I think we are missing `Letter` Design? https://github.com/guardian/types/blob/main/src/format.ts#L24

Reply:
That's a good question. We don't currently have `Letter` as a `Design` in our [`decideDesign.ts`](https://github.com/guardian/dotcom-rendering/blob/main/src/web/lib/decideDesign.ts) function.
@oliverlloyd do you know if whether `@guardian/types` is correct or if DCR is?

Reply:
`@guardian/types` trumps DCR. We haven't yet updated the version of the /types package we're using but when we do we will add support for `Letter`

Reply:
Letter is a recent addition: https://github.com/guardian/types/pull/69